java/adapter/avro/src/main/java/org/apache/arrow/consumers/AvroBytesConsumer.java

```diff
@@ -43,12 +44,15 @@ public AvroBytesConsumer(VarBinaryVector vector) {
   @Override
   public void consume(Decoder decoder) throws IOException {
     VarBinaryHolder holder = new VarBinaryHolder();
-    ByteBuffer byteBuffer = decoder.readBytes(null);
+
+    //cacheBuffer is initialized null and create in the first consume,
```

Comment:
nit: space after //

Reply:
Thanks for reminder, seems we should enhance style check for something like continues empty line which could speed up code review :)

Reply:
if that is possible it would be good to have.